packages/@coorpacademy-components/src/molecule/setup-cohort-item/index.js

```diff
@@ -32,15 +32,18 @@ const InputSplitScreen = props => {
     <div>
       <div className={style.splitDefault}>
         <div className={style.splitLeft}>
-          <p className={style.title}>{leftSection.title}</p>
+          <p className={style.title}>{leftSection.title}*</p>
```

Comment:
include a `mandatory` (or  `required`) field on props for manage `*`

Reply:
done